### PR TITLE
WIP - Fix merge_holding_item_info to include enum and chron

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: lib/voyager_helpers/liberator.rb [hunk 1]
```diff
@@ -796,11 +796,20 @@ def merge_holding_item_into_bib(bib, holding, item, recap=false, conn=nil)
...
             record_hash['fields'] << h
           end
+          item_enum_chron = ''
+          if item[:enum]
+            item_enum_chron << "#{item[:enum]} "
+          end
+          if item[:chron]
+            item_enum_chron << "(#{item[:chron]})"
+          end
+          item_enum_chron.gsub!(/^(.*)[ ]+$/, '\1')
```
Comment (tampakis) [inline lib/voyager_helpers/liberator.rb]: I think it might be preferable use the logic on [these lines](https://github.com/pulibrary/voyager_helpers/blob/master/lib/voyager_helpers/liberator.rb#L252-L257) for getting the enum/chron string. 

Also can you add a spec for when both enum and chron are nil, as well as when one of them is nil? I think that when there is just the chron present that we wouldn't want the value to be surrounded by parentheses.

Comment (kevinreiss) [inline lib/voyager_helpers/liberator.rb]: I don't think we've addressed these comments yet @tampakis. We should wrap this one up next week.

File: lib/voyager_helpers/liberator.rb [hunk 2]
```diff
@@ -782,25 +782,33 @@ def merge_holding_item_into_bib(bib, holding, item, recap=false, conn=nil)
...
+          item_enum_chron = item[:enum]
+          unless item[:chron].nil?
+            item_enum_chron << " (#{item[:chron]})"
+          end
+        elsif item[:chron]
+          item_enum_chron = "#{item[:chron]}"
         end
         if recap && holding_location =~ /^rcp[a-z]{2}$/
+          call_no = holding['852']['h'] unless holding['852']['h'].nil?
+          call_no << " #{holding['852']['i']}" unless holding['852']['i'].nil?
```
Comment (tampakis) [inline lib/voyager_helpers/liberator.rb]: This will error if subfield h is nil.

File: lib/voyager_helpers/liberator.rb [hunk 3]
```diff
@@ -782,25 +782,33 @@ def merge_holding_item_into_bib(bib, holding, item, recap=false, conn=nil)
...
-        if holding['852']['i']
-          combined_call_no = "#{holding['852']['h']} #{holding['852']['i']}"
+        item_enum_chron = nil
+        if item[:enum]
+          item_enum_chron = item[:enum]
+          unless item[:chron].nil?
+            item_enum_chron << " (#{item[:chron]})"
+          end
+        elsif item[:chron]
+          item_enum_chron = "#{item[:chron]}"
```
Comment (tampakis) [inline lib/voyager_helpers/liberator.rb]: Not a big deal, but the string interpolation isn't necessary here.

File: spec/unit/voyager_helpers/liberator_spec.rb [hunk 4]
```diff
@@ -349,28 +375,53 @@ def json_stub(filename)
...
           full_record = described_class.send(:merge_holding_item_into_bib, recap_bib_record, recap_mfhd_record, recap_item_info, recap=true)
           expect(full_record['852']['h']).to eq 'BM899.61 .A39 M8'
           expect(full_record['876']['0']).to eq recap_mfhd_id
+          expect(full_record['876']['3']).to eq 'v.2 (1948)'
+          expect(full_record['876']['a']).to eq recap_item_id
+          expect(full_record['876']['j']).to eq 'Not Charged'
           expect(full_record['876']['p']).to eq recap_barcode
+          expect(full_record['876']['t']).to eq '1'
           expect(full_record['876']['x']).to eq 'Shared'
+
```
Comment (tampakis) [inline spec/unit/voyager_helpers/liberator_spec.rb]: unnecessary whitespace

**Comment by mzelesky** [comment]
@tampakis I addressed the issues you identified above. Could you please review again?

Thank you.

**Comment by tampakis** [comment]
Cool
